TMGToolbox/src/input_output/import_binary_matrix.py

```diff
@@ -221,12 +230,18 @@ def _Execute(self):
             else:
                 data = _MatrixData.load(self.ImportFile)
             
-            origins, destinations = data.indices
-            origins = set(origins)
-            destinations = set(destinations)
-            if origins ^ destinations:
-                raise Exception("Asymmetrical matrix detected. Matrix must be square.")
-            
+            # 2D matrix
+            if self.MatrixType in ['mf']:
```

Comment:
If the self.MatrixType is a string variable, it is kind of odd to check an list of 1 item instead of just checking the single item directly for equality.